CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.251 Jan 20 2023
+- Update `NodePool` with status attributes
```

Comment:
Added "current_compute" attribute in `ClusterStatus` for hosted clusters.

Reply:
instead or with what is there?

Reply:
No, sorry, like a new line.
Your line is good.